### DISCUSSION TITLE
/opt/keyfactor/wildfly-26.1.3.Final/bin/standalone.sh abort starting the container

### DISCUSSION THREAD

**Discussion by oudehomar** [Keyfactor/ejbca-ce#634] [Q&A] [answered]
Hi, 

i am on Mac 14.5 with M2 Chip. I have a problem starting jebca and signserver  in the container. i get the error : 

<img width="1249" alt="Screenshot 2024-07-21 at 10 20 30" src="https://github.com/user-attachments/assets/4356a893-cda4-4ba4-b7e5-732de8a81222">

my java version is :
`$ java --version 
openjdk 11.0.23 2024-04-16 LTS
OpenJDK Runtime Environment (build 11.0.23+10-LTS)
OpenJDK 64-Bit Server VM (build 11.0.23+10-LTS, mixed mode)`

i start the docker container using : 
`docker run --name ejbcakey -p 8822:8443  keyfactor/ejbca-ce:8.3.2`

I had no problem running EJBCA version 8.0.0 two months ago. Now, I get the same error with all EJBCA versions.

Any help about this error? i could not find any similar discussion in the community.
Thanks,

**Comment by primetomas** [comment, marked as answer]
ARM builds of the containers are not yet available, only i64. That is probably why. I believe you could run it in a VM with Ubuntu or something. I think ARM containers will be available in the future sometime,
